README.md

```diff
@@ -100,31 +94,51 @@ should work on LX-branded zones as well.
     deployment (see
     [the CNS operator guide](https://github.com/joyent/triton-cns/blob/master/docs/operator-guide.md)).
     We'll assume for the sake of example here that the CNS suffix for the
-    DC is `us-west-1.triton.zone`.
+    DC is
+    ```
+    us-west-1.triton.zone
+    ```
  2. Find the CNS-generated name for your container. One way to do this is
-    to look at the output of `triton inst get <instance>` for the `dns_names`
-    array. As an example, let's consider
-    `blog.svc.3c330096-89e6-11e7-9f13-23d71a63353e.us-west-1.triton.zone`.
+    to look for the `dns_names` array in the output of
+    ```shell
```

Comment:
This line and elsewhere will not render properly.

* fenced code blocks need to have a blank line above and below
* code blocks that should be part of the preceding list item should be indented an additional 4 spaces

Use markdownlint to verify markdown syntax.